### PR TITLE
Minor fixes to spacksites setup

### PR DESCRIPTION
**Pull request by giordano**

### CODE REVIEW

File: spacksites/process-env-scripts/spack-deps-rhel-7.8.sh [hunk 1]
```diff
@@ -11,11 +11,11 @@
 #
 # This code was contributed by Owain Kenway
 
-UCL_SCLS="rh-python38 devtoolset-11"
```
Comment (giordano) [inline spacksites/process-env-scripts/spack-deps-rhel-7.8.sh]: In `/opt/rh` I only have/see `devtoolset-11`.

Comment (heatherkellyucl) [reply spacksites/process-env-scripts/spack-deps-rhel-7.8.sh]: This is an issue on login12 - it exists everywhere else.

**Review by heatherkellyucl — CHANGES_REQUESTED** [review]
The other changes are fine, but `UCL_SCLS="rh-python38 devtoolset-11"` should not change.

File: spacksites/process-env-scripts/spack-deps-rhel-7.8.sh (hunk 1 above)
Comment (giordano) [reply spacksites/process-env-scripts/spack-deps-rhel-7.8.sh]: Ah, wow, looks like I got the lucky node :slightly_smiling_face:  Fixed with latest push, thanks!

**Review by heatherkellyucl — APPROVED** [review]

**Pull request merged**